src/phoenix/server/templates/index.html

```diff
@@ -11,7 +51,6 @@
     <meta property="og:description" content="AI Observability & Evaluation" />
     <meta property="og:image" content="https://raw.githubusercontent.com/Arize-ai/phoenix-assets/main/images/socal/social-preview-horizontal.jpg" />
     <meta name="theme-color" content="#ffffff" />
-    <link rel="stylesheet" src="{{basename}}/index.css"></link>
```

Comment:
why is the scripts not in the HEAD?

Reply:
moved css into head and kept js in body to not block page loading on js bundles